x/pool-incentives/keeper/keeper_test.go

```diff
@@ -96,62 +96,103 @@ func (suite *KeeperTestSuite) TestCreateConcentratePoolGauges() {
 	}
 }
 
-func (suite *KeeperTestSuite) TestCreatePoolGauges() {
+func (suite *KeeperTestSuite) TestCreateLockablePoolGauges() {
 	tests := []struct {
-		name        string
-		poolId      uint64
-		poolType    poolmanagertypes.PoolType
-		expectedErr bool
+		name                   string
+		poolId                 uint64
+		expectedGaugeDurations []time.Duration
+		expectedGaugeIds       []uint64
+		expectedErr            bool
 	}{
 		{
-			name:        "Concentrated Liquidity Pool",
-			poolId:      1,
-			poolType:    poolmanagertypes.Concentrated,
-			expectedErr: false,
+			name:                   "Create Gauge with valid PoolId",
+			poolId:                 uint64(1),
+			expectedGaugeDurations: []time.Duration{time.Hour, time.Hour * 3, time.Hour * 7},
+			expectedGaugeIds:       []uint64{4, 5, 6}, //note: it's not 1,2,3 because we create 3 gauges during setup of suite.PrepareBalancerPool()
+			expectedErr:            false,
 		},
 		{
-			name:        "non concentrated pool",
-			poolId:      2,
-			poolType:    poolmanagertypes.Balancer,
-			expectedErr: false,
-		},
-		{
-			name:        "non existent pool",
-			poolId:      0,
-			expectedErr: true,
+			name:                   "Create Gauge with invalid PoolId",
+			poolId:                 uint64(0),
+			expectedGaugeDurations: nil,
+			expectedGaugeIds:       []uint64{},
+			expectedErr:            true,
 		},
 	}
 
 	for _, tc := range tests {
 		suite.Run(tc.name, func() {
 			suite.SetupTest()
-			suite.PrepareConcentratedPool().GetId()
 			suite.PrepareBalancerPool()
 
-			var err error
-			// TODO: split into separate tests
+			err := suite.App.PoolIncentivesKeeper.CreateLockablePoolGauges(suite.Ctx, tc.poolId)
+			if tc.expectedErr {
+				suite.Require().Error(err)
+			} else {
+				suite.Require().NoError(err)
+
+				for idx, duration := range tc.expectedGaugeDurations {
```

Comment:
As a sanity check and to avoid false positives, please add  an assertion that `tc.expectedGaugeDurations` is not empty